src/.nuget/toolchain.ubuntu.14.04-x64.Microsoft.DotNet.ILCompiler.Development.nuspec

```diff
@@ -32,6 +32,8 @@
       <file src="../ToolRuntime/*.dll" />
```

Comment:
I know this has already been here, but we're sucking up a random build artifact into the Nuget package here. ToolRuntime is just some version of CoreCLR that is there to support the build. It doesn't mean it has been tested or blessed to be distributed with the product. The version might even change any time without warning, because it's whatever the Microsoft.DotNet.BuildTools package brings down.